src/stable/lib/globals.ts

```diff
@@ -12,42 +12,51 @@ import { Action } from './state';
 
 declare global {
     // eslint-disable-next-line no-var
-    var _azleActions: Action[];
+    var _azleActions: Action[] | undefined;
     // eslint-disable-next-line no-var
     var _azleCanisterClassMeta: CanisterClassMeta | undefined;
     // eslint-disable-next-line no-var
-    var _azleCanisterMethodNames: { [key: string]: string };
+    var _azleCanisterMethodNames: { [key: string]: string } | undefined;
     // eslint-disable-next-line no-var
-    var _azleDispatch: (action: Action) => void;
+    var _azleDispatch: ((action: Action) => void) | undefined;
     // eslint-disable-next-line no-var
-    var _azleExperimental: boolean;
+    var _azleExperimental: boolean | undefined;
     // eslint-disable-next-line no-var
     var _azleIcExperimental: AzleIcExperimental | undefined;
     // eslint-disable-next-line no-var
-    var _azleIcpReplicaWasmEnvironment: boolean;
+    var _azleIcpReplicaWasmEnvironment: boolean | undefined;
     // eslint-disable-next-line no-var
     var _azleIc: AzleIc | undefined;
     // eslint-disable-next-line no-var
-    var _azleInitCalled: boolean;
+    var _azleInitCalled: boolean | undefined;
     // eslint-disable-next-line no-var
-    var _azleNodejsWasmEnvironment: boolean;
+    var _azleNodejsWasmEnvironment: boolean | undefined;
     // eslint-disable-next-line no-var
-    var _azlePostUpgradeCalled: boolean;
+    var _azlePostUpgradeCalled: boolean | undefined;
```

Comment:
same comments in same places as with _azleInitCalled

Reply:
I think I have addressed those.